src/test/java/com/techsophy/tsf/workflow/utils/TokenUtilsPaginationTest.java

```diff
@@ -94,6 +95,7 @@ void getLoggedInUserIdTest()
     void getSortByTest()
     {
         String[] dogs = {"Pitbull", "Poodle", "Lab", "Pug"};
-        tokenUtils.getSortBy(dogs);
+        Sort response = tokenUtils.getSortBy(dogs);
```

Comment:
Write proper test case to check the conditions based on input data or remove this test case.